ci-operator/config/openshift/jenkins-sync-plugin/openshift-jenkins-sync-plugin-release-4.9.yaml

```
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.10
+    tag: golang-1.13
```

Update to use golang 1.16, no?

```suggestion
    tag: golang-1.16
```